modules/system/lang/vn/validation.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+return [
+
```

Comment:
Don't include this file if it doesn't have any translations please

Reply:
I have compiled it